.github/workflows/sycl-docs.yml

```diff
@@ -25,16 +25,14 @@ jobs:
         path: repo
     - name: Install deps
       run: |
-        sudo apt-get install -y doxygen graphviz ssh ninja-build libhwloc-dev
+        sudo apt-get install -y graphviz ssh ninja-build libhwloc-dev
         sudo pip3 install -r repo/llvm/docs/requirements.txt
     - name: Build Docs
       run: |
         mkdir -p $GITHUB_WORKSPACE/build
         cd $GITHUB_WORKSPACE/build
         python $GITHUB_WORKSPACE/repo/buildbot/configure.py -w $GITHUB_WORKSPACE \
         -s $GITHUB_WORKSPACE/repo -o $GITHUB_WORKSPACE/build -t Release --docs
```

Comment:
Potentially it can be useful to verify that developers can build Doxygen documentation locally if needed. Currently, we build Doxygen documentation during pre-commit testing w/o deploying the documentation.

I don't have a strong opinion about keeping Doxygen documentation buildable. IMHO, even if broken, it should be easy to fix.
If any of code owners have an opinion, feel free to leave a comment.